flink-core/src/main/java/org/apache/flink/util/StringUtils.java

```diff
@@ -348,6 +350,21 @@ public static String concatenateWithAnd(@Nullable String s1, @Nullable String s2
 		}
 	}
 
+	/**
+	 * Generates a string containing a comma-separated list of values in double-quatas.
+	 * Uses lower-cased values returning from {@link Object#toString()} method for each element in the given array.
+	 * Null values are spipped.
+	 *
+	 * @param values array of elements for the list
+	 *
+	 * @return The string with quated list of elements
+	 */
+	public static String toQuatedListString(Object[] values) {
```

Comment:
We could move this into the `Utils´ class in `flink-docs`, at the moment I don't see much other use for it..